app/components/TopBar.tsx

```diff
@@ -38,7 +38,7 @@ export function TopBar({ children }: { children: React.ReactNode }) {
       {/* Height is governed by PageContainer grid */}
       {/* shrink-0 is needed to prevent getting squished by body content */}
       <div className="z-topBar border-b bg-default border-secondary">
-        <div className="mx-3 flex h-[60px] shrink-0 items-center justify-between">
+        <div className="mx-3 flex h-[54px] shrink-0 items-center justify-between">
```

Comment:
Was going to ask if it was worth making this a var, but that might only be worthwhile if we need to use it in more places.

Reply:
WELP

https://github.com/oxidecomputer/console/blob/b4920b1157c5f63a8d5a39adcdb62c541ca870ec/app/pages/project/instances/instance/SerialConsolePage.tsx#L138

Reply:
womp

Reply:
I dropped this down from 60 to 54, but this is technically for delineating between the body and the footer. @benjaminleonard do we want the serial console footer to match the header, at 54px? Or to stay at 60?

Reply:
I think in that case it's making the layout the full height of the screen, minus the size of the top bar. The bottom bar on the serial console seems a bit chunky but it's matching the instance create bottom bar. Might adjust elsewhere.